all_available_tdts.js

```diff
@@ -5,24 +5,31 @@ const TBTCDepositToken = require("@keep-network/tbtc/artifacts/TBTCDepositToken.
 const Deposit = require("@keep-network/tbtc/artifacts/Deposit.json");
 const VendingMachine = require("@keep-network/tbtc/artifacts/VendingMachine.json");
 
+const config = require('./config')
+const infura = config.infura_secret || process.env.INFURA_API;
+const network = config.network || 'homestead'
```

Comment:
Switching between mainnet and testnet is not as easy; the ABIs of the smart contracts might differ, so we cannot just change like that. In addition, all the `networks["1"]` would need to be changed to `networks["3"]`.

Reply:
oh true, good catch.  I will revert that and recommit.